src/wp-includes/script-loader.php

```diff
@@ -1843,6 +1843,32 @@ function wp_just_in_time_script_localization() {
 	);
 }
 
+
+/**
+ * Prints a loader script if there is text/template registered script.
+ *
+ * When added to the DOM, this script converts any text/template scripts
+ * associated with a handle into type/javascript, and executes them.
+ */
+function wp_print_template_loader_script() {
+	$wp_scripts = wp_scripts();
+
+	if ( $wp_scripts->has_delayed_inline_script() ) {
+		$output    = <<<JS
+function wpLoadAfterScripts( handle ) {
+	let scripts = document.querySelectorAll(`[type="text/template"][data-wp-executes-after="\${handle}"]`);
+	scripts.forEach( (script) => {
+		script.setAttribute("type","text/javascript");
+		eval(script.innerHTML);
```

Comment:
Since only text is contained in a `script`, I think `textContent` is more appropriate than `innerHTML`, although they may behave the same in the end.

```suggestion
		eval(script.textContent);
```

Reply:
@westonruter I've addressed this in a recent commit, thanks. I ran a few tests and for the purposes of inline scripts it seems totally fine, although as you correctly indicated the outcome was identical.